Makefile

```diff
@@ -97,8 +100,13 @@ sub-push-%: container-% do-push-% ;
 
 # do build and then push all official images
 push: gcr-login $(addprefix sub-push-,$(ALL_ARCHITECTURES)) ;
-	# TODO: push with manifest-tool?
-	# Should depend on target: ./manifest-tool
+
+all-push: push push-manifest
```

Comment:
Maybe the all-push target could be before push and push-manifest ?
It seems odd to have one declared after, and one before.

Reply:
Done!